integration_tests/tests/flowmachine_server_tests/test_action_run_query.py

```diff
@@ -65,12 +67,12 @@
     #
 
     first_few_rows_expected = [
-        ("524 3 09 50", 18),
-        ("524 5 13 67", 17),
+        ("524 1 01 04", 13),
```

Comment:
Why do we expect these values to change? Shouldn't the internal calculation still yield the same result?

Reply:
They do! But they are now in a consistent order, so the first three rows are different.